sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
if 2 makes you happy go for it :-)
Just a commentary.... 
1) I don't like tools telling me how to structure my code - if I want one HUGE dir, let me
2) I don't expect this cli to be anything more than a test tool so I can't get too worked up over sticking to a formal (long-term-pretty) structure, and on the flip side, its the reason I wanted to keep it simple and in one dir.